nap/engine.py

```diff
@@ -74,7 +75,7 @@ def _generate_url(self, url_type='lookup', resource_obj=None, **kwargs):
             }
 
             url_match_vars = dict([
-                (k, v) for (k, v) in model_keywords.items()
+                (k, v) for (k, v) in list(model_keywords.items())
```

Comment:
Yeah, this was auto updated. Let's just ditch the list. The differences is
that in py2 .items is a list and in py3 it's an iterator, but that doesn't
matter here.

On Fri, Nov 13, 2015 at 9:21 AM, Joseph Bane <notifications@github.com>
wrote:

> In nap/engine.py
> <https://github.com/CashStar/nap/pull/9#discussion_r44786195>:
>
> > @@ -74,7 +75,7 @@ def _generate_url(self, url_type='lookup', resource_obj=None, **kwargs):
> >              }
> >
> >              url_match_vars = dict([
> > -                (k, v) for (k, v) in model_keywords.items()
> > +                (k, v) for (k, v) in list(model_keywords.items())
>
> Is there a performance improvement forcing this from an iterator to a list
> in a comprehension?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CashStar/nap/pull/9/files#r44786195>.
>